### PR TITLE
ENTITY-2960 - Refactor Environment Property Keys

### PR DESCRIPTION
**Pull request by bsnopek-freshworks**
*Issue #:* /bcgov/entity#2960

*Description of changes:*
- Updated configurtion.json to rename AUTH_API_URL to VUE_APP_AUTH_ROOT_API
- Updated k8s/templates to reflect changes
- Updated config-helper to reflect changes
- Updated release / deployment checklist to account for changes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/utils/config-helper.ts [hunk 1]
```diff
@@ -42,10 +42,9 @@ export function fetchConfig (): Promise<any> {
 
       // TODO: There should be a bit of a cleanup here to ensure these use the same prop-name.
       // -- VUE_APP_AUTH_ROOT_API is expected by sbc-common-components
-      const authApiUrl: string = response.data['AUTH_API_URL']
-      sessionStorage.setItem('AUTH_API_URL', authApiUrl)
-      sessionStorage.setItem('VUE_APP_AUTH_ROOT_API', response.data['VUE_APP_AUTH_ROOT_API'])
-      console.log('Set Auth API URL to: ' + authApiUrl)
+      const authApiUrl: string = response.data['VUE_APP_AUTH_ROOT_API']
+      sessionStorage.setItem('VUE_APP_AUTH_ROOT_API', authApiUrl)
+      console.log('Set Auth VUE_APP_AUTH_ROOT_APIL to: ' + authApiUrl)
```
Comment (severinbeauvais) [inline src/utils/config-helper.ts]: Typo.
Also I suggets just leaving the old message (since it's still valid).

**Comment by severinbeauvais** [comment]
Don't merge this yet. I believe there are other changes coming.

File: public/config/configuration.json [hunk 2]
```diff
@@ -2,7 +2,6 @@
   "API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/",
   "AUTH_URL": "https://dev.bcregistry.ca/cooperatives/auth/",
   "SIGNIN_URL": "https://dev.bcregistry.ca/cooperatives/auth/signin/bcsc/",
-  "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/",
```
Comment (severinbeauvais) [inline public/config/configuration.json]: You replaced this key with the new one, but this project still needs this value. See https://github.com/bcgov/bcrs-business-create-ui/blob/b6da247b368cf2b35aa1f0bed5ecc76c9ea68081/src/utils/auth-helpers.ts#L86

Comment (bsnopek-freshworks) [reply public/config/configuration.json]: done

**Pull request merged**